<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite errors="1" failures="1" hostname="E0004929GM" name="pytest" skipped="0" tests="24" time="1308.811" timestamp="2021-07-22T19:06:56.319141"><testcase classname="test_dir.test_a_kernel.TestKernel" name="test_sanity_2" time="123.266" /><testcase classname="test_dir.test_a_kernel.TestKernel" name="test_sanity_8" time="16.621" /><testcase classname="test_dir.test_c_videoplayback.TestVideoplayback" name="test_sanity_15" time="55.945" /><testcase classname="test_dir.test_c_videoplayback.TestVideoplayback" name="test_sanity_16" time="35.228" /><testcase classname="test_dir.test_c_videoplayback.TestVideoplayback" name="test_sanity_17" time="33.555" /><testcase classname="test_dir.test_c_videoplayback.TestVideoplayback" name="test_sanity_18" time="33.288" /><testcase classname="test_dir.test_c_videoplayback.TestVideoplayback" name="test_sanity_19" time="34.350" /><testcase classname="test_dir.test_c_videoplayback.TestVideoplayback" name="test_sanity_20" time="35.105" /><testcase classname="test_dir.test_c_videoplayback.TestVideoplayback" name="test_sanity_21" time="59.829" /><testcase classname="test_dir.test_c_videoplayback.TestVideoplayback" name="test_sanity_22" time="60.234" /><testcase classname="test_dir.test_d_musicplayback.TestMusicplayback" name="test_sanity_23" time="55.573" /><testcase classname="test_dir.test_d_musicplayback.TestMusicplayback" name="test_sanity_24" time="50.908" /><testcase classname="test_dir.test_d_musicplayback.TestMusicplayback" name="test_sanity_25" time="34.132" /><testcase classname="test_dir.test_d_musicplayback.TestMusicplayback" name="test_sanity_26" time="101.143"><failure message="assert False is True">self = &lt;sanity_ld60_mini.test_dir.test_d_musicplayback.TestMusicplayback object at 0x000001FC19F53F98&gt;
driver = &lt;appium.webdriver.webdriver.WebDriver (session="b1225b08-00eb-464a-aafd-1de9d2a54200")&gt;

    def test_sanity_26(self, driver):
        page = MusicPage(driver)
        page.click_music_26()
        result = page.playback_music(*page.music_26_name)
&gt;       assert result is True
E       assert False is True

test_dir\test_d_musicplayback.py:38: AssertionError</failure></testcase><testcase classname="test_dir.test_d_musicplayback.TestMusicplayback" name="test_sanity_27" time="60.464" /><testcase classname="test_dir.test_d_musicplayback.TestMusicplayback" name="test_sanity_28" time="53.885" /><testcase classname="test_dir.test_d_musicplayback.TestMusicplayback" name="test_sanity_29" time="36.169" /><testcase classname="test_dir.test_e_pictureplayback.TestPictureplayback" name="test_sanity_30" time="34.740" /><testcase classname="test_dir.test_e_pictureplayback.TestPictureplayback" name="test_sanity_31_1" time="17.846" /><testcase classname="test_dir.test_f_audiooutput.TestAudio" name="test_sanity_36" time="153.781" /><testcase classname="test_dir.test_f_audiooutput.TestAudio" name="test_sanity_37" time="130.466" /><testcase classname="test_dir.test_mediascanner.TestMediaScanner" name="test_sanity_61" time="37.496" /><testcase classname="test_dir.test_mediascanner.TestMediaScanner" name="test_sanity_62" time="13.873" /><testcase time="39.242" /><testcase classname="pytest" name="internal" time="0.000"><error message="internal error">Traceback (most recent call last):
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\_pytest\main.py", line 269, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\_pytest\main.py", line 323, in _main
    config.hook.pytest_runtestloop(session=session)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\manager.py", line 87, in &lt;lambda&gt;
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\callers.py", line 208, in _multicall
    return outcome.get_result()
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\_pytest\main.py", line 348, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\manager.py", line 87, in &lt;lambda&gt;
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\callers.py", line 208, in _multicall
    return outcome.get_result()
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\_pytest\runner.py", line 109, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\_pytest\runner.py", line 126, in runtestprotocol
    reports.append(call_and_report(item, "call", log))
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\_pytest\runner.py", line 217, in call_and_report
    report: TestReport = hook.pytest_runtest_makereport(item=item, call=call)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\manager.py", line 87, in &lt;lambda&gt;
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\pluggy\callers.py", line 203, in _multicall
    gen.send(outcome)
  File "C:\Python_tools\sanity_ld60_mini\conftest.py", line 69, in pytest_runtest_makereport
    capture_screenshots(case_name)
  File "C:\Python_tools\sanity_ld60_mini\conftest.py", line 117, in capture_screenshots
    driver.get_screenshot_as_file(image_dir)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\appium\webdriver\errorhandler.py", line 31, in check_response
    raise wde
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\appium\webdriver\errorhandler.py", line 26, in check_response
    super().check_response(response)
  File "C:\Python_tools\sanity_ld60_mini\venv\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started</error></testcase></testsuite></testsuites>